bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/actions/NotificationActionTypeProvider.java

```diff
@@ -34,41 +34,62 @@
-            getSendNotificationConfig(false, null), "send a notification",
+            getNotificationConfig(ConfigType.NOT_EXTENDED, true, null), "send a notification",
             "Sends a notification to a specific cloud user.", null, Visibility.VISIBLE, null, null);
+
     private static final ModuleType SEND_EXTENDED_NOTIFICATION_ACTION = new ActionType(
-            SendNotificationActionHandler.EXTENDED_TYPE_ID, getSendNotificationConfig(true, null),
+            SendNotificationActionHandler.EXTENDED_TYPE_ID, getNotificationConfig(ConfigType.EXTENDED, true, null),
             "send a notification with icon and severity",
             "Sends a notification to a specific cloud user. Optionally add an icon or the severity.", null,
             Visibility.VISIBLE, null, null);
+
+    private static final ModuleType SEND_EXTENDED2_NOTIFICATION_ACTION = new ActionType(
+            SendNotificationActionHandler.EXTENDED2_TYPE_ID, getNotificationConfig(ConfigType.EXTENDED2, true, null),
+            "send a notification with icon, severity, on click, media attachment, action button 1, action button 2, action button 3",
+            "Sends a notification to a specific cloud user. Optionally add an icon, severity, on click action, media to attach, and up to 3 action buttons with a format of \"Title=Action\".",
+            null, Visibility.VISIBLE, null, null);
+
     private static final ModuleType SEND_BROADCAST_NOTIFICATION_ACTION = new ActionType(
-            SendBroadcastNotificationActionHandler.TYPE_ID, getNotificationConfig(false, null),
+            SendBroadcastNotificationActionHandler.TYPE_ID, getNotificationConfig(ConfigType.NOT_EXTENDED, true, null),
             "broadcast a notification", "Sends a notification to all devices of all users.", null, Visibility.VISIBLE,
             null, null);
-    private static final ModuleType SEND_EXRENDED_BROADCAST_NOTIFICATION_ACTION = new ActionType(
-            SendBroadcastNotificationActionHandler.EXTENDED_TYPE_ID, getNotificationConfig(true, null),
-            "broadcast a notification with icon and severity",
+
+    private static final ModuleType SEND_EXTENDED_BROADCAST_NOTIFICATION_ACTION = new ActionType(
+            SendBroadcastNotificationActionHandler.EXTENDED_TYPE_ID,
+            getNotificationConfig(ConfigType.EXTENDED, false, null), "broadcast a notification with icon and severity",
             "Sends a notification to all devices of all users. Optionally add an icon or the severity.", null,
             Visibility.VISIBLE, null, null);
+
+    private static final ModuleType SEND_EXTENDED2_BROADCAST_NOTIFICATION_ACTION = new ActionType(
+            SendBroadcastNotificationActionHandler.EXTENDED2_TYPE_ID,
+            getNotificationConfig(ConfigType.EXTENDED2, false, null),
+            "broadcast a notification with with icon, severity, on click, media attachment, action button 1, action button 2, action button 3",
```

Comment:
Looks better in the UI:

```suggestion
            "broadcast a notification with with icon, severity, on click, media attachment, action buttons",
```